op-guide/Monitoring_TiDB.md

```diff
@@ -0,0 +1,183 @@
+# Monitoring a TiDB cluster
```

Comment:
change file name like `monitoring-tidb.md`  to  consistent with others in op-guide